src/processors/README.md

```diff
@@ -40,6 +40,20 @@ Example:
 
 No config needed.
 
+### filter-when-format
+
+`filter-when-format` will check if the first docblock comment of the file contains a `@format` tag. That means that the file was formatted with Prettier or a similar tool and that formatting-only rules (like `indent`) can be ignored for this file. The motivation for this processor is that ESLint rules sometimes have a different opinion about formatting than Prettier, so we want to conditionally turn them of and treat Prettier as the authority.
```

Comment:
Updated the README with your wording. I just changed the past tense to present tense -- I think the rule will be actively used for a very long time before we fully migrate the Calypso codebase. The past tense suggests that it is a legacy rule that we used to need in very distant past. :)